projects/schematics/src/add-spartacus/schema.json

```diff
@@ -47,7 +47,8 @@
         "Store-Finder",
         "Personalization",
         "Account",
-        "Profile"
+        "Profile",
+        "Checkout"
```

Comment:
Can we also change order of Checkout in the prompts, so we still keep it alphabetical?